src/main/java/org/spongepowered/api/entity/living/player/server/ServerPlayer.java

```diff
@@ -128,29 +128,37 @@ default boolean isViewingInventory() {
      *
      * @return The player's view distance
      */
-    int getViewDistance();
+    default int getViewDistance() {
+        return this.require(Keys.VIEW_DISTANCE);
+    }
 
     /**
      * Sets the view distance setting of the player. This value represents the
      * radius (around the player) in unit chunks.
      *
      * @param distance The player's view distance
      */
-    void setViewDistance(int distance);
+    default void setViewDistance(final int distance) {
```

Comment:
I see you were just updating this, but I also see that this view distance key is documented as read only. So I would imagine we shouldn’t be able to set a player’s view distance here as well.

Reply:
yeah -- this api method was unimplemented, but I preserved it since it existed, and documented what's currently easily implementable. Can update either way if I get confirmation on what'll be implemented.

Reply:
@zml2008 Additionally, data methods follow a certain style. Look at `Entity` to see.